android/build.gradle

```diff
@@ -17,8 +17,8 @@ buildscript {
         kotlin_version = "1.3.50"
         supportLibVersion = "28.0.0"
         libre_build = !(isPlay.toBoolean())
-        jitsi_url = isPlay ? "https://github.com/RocketChat/jitsi-maven-repository/raw/master/releases" : "https://github.com/RocketChat/jitsi-maven-repository/raw/libre/releases"
-        jitsi_version = isPlay ? "2.10.2" : "2.10.0-libre"
+        jitsi_url = isPlay ? "http://github.com/RocketChat/jitsi-maven-repository/raw/3.7.0/releases" : "https://github.com/jitsi/jitsi-maven-repository/raw/libre/releases"
```

Comment:
We're not actively maintaining libre, so I just changed the URL to match, but there's no point to test at this time.